src/main/java/randoop/generation/GrtFuzzer.java

```diff
@@ -27,7 +27,10 @@ public GrtFuzzer() {}
   /** List of all fuzzers. */
   @SuppressWarnings("ClassInitializationDeadlock")
   private static final List<GrtFuzzer> FUZZERS =
-      Arrays.asList(GrtNumericFuzzer.getInstance(), GrtStringFuzzer.getInstance());
+      Arrays.asList(
+          GrtNumericFuzzer.getInstance(),
+          GrtStringFuzzer.getInstance(),
+          GrtObjectFuzzer.getInstance());
```

Comment:
Does this need to be instantiated so early?  The initialization code in `ForwardGenerator` is a bit complex and feels out of place.  Is there some other location where the fuzzers can be instantiated, so that the invocations of the fuzzer in `ForwardGenerator` don't have to possibly do initalization?